ts/services.ts

```diff
@@ -1,22 +1,38 @@
 import { closeEventDetailsModal } from "./modal.js";
 import { loadEventsForCurrentWeek } from "./events.js";
-import { Event} from './types.js'
-export function saveEventToLocalStorage(event:Event): void {
-  let events: Event[] = JSON.parse(localStorage.getItem("calendarEvents") || '[]');
-  events.push(event);
-  localStorage.setItem("calendarEvents", JSON.stringify(events));
-}
+import { Event} from './types.js';
+
+const API_URL = "http://localhost:3000/events"
 
-export function getEventsFromLocalStorage(): Event[] {
-  let events: Event[] = JSON.parse(localStorage.getItem("calendarEvents") || '[]');
-  return events;
+export const saveEventToServer = async (event:Event): Promise<Event> =>  {
+  const response = await fetch(API_URL, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(event),
+  });
+  if (!response.ok) {
```

Comment:
I don't think this is correct way to deal with errors, since this approach only deals with not successful status code http errors. But what about network errors or what if response is not a valid json and .json throws an error? 

why not try catch?